lightly_studio/tests/core/test_add_samples.py

```diff
@@ -14,11 +16,16 @@
 from PIL import Image as PILImage
 from sqlmodel import Session
 
+from lightly_studio import Dataset
 from lightly_studio.core import add_samples
 from lightly_studio.models.image import ImageCreate
 from lightly_studio.models.sample import SampleTable
 from lightly_studio.resolvers import caption_resolver, image_resolver
-from tests.helpers_resolvers import create_dataset
+from tests.helpers_resolvers import (
+    ImageStub,
+    create_dataset,
+    create_images,
+)
```

Comment:
Code style: We don't import functions directly, but rather their file. Then call `helpers_resolver.create_dataset` and `helpers_resolvers.create_image`. 

```suggestion
from tests.helpers_resolvers import (
    ImageStub,
)
from tests import helpers_resolvers
```